wordpress-mobile/WordPress-FluxC-Android
# Woo: Delete coupon functionality

Fixes #2230.

**To test:**
1. Start the example app
2. Go to Woo -> Coupons
3. Fetch the coupons and find the ID of a coupon to delete
4. Tap on the Delete coupon button and enter a valid coupon ID
5. Verify the coupon is successfully deleted
